### ISSUE TITLE
Conflicting Reproduction Commands

### ISSUE THREAD

**Issue by inboxedshoe** [isaykatsman/Shedding-Light-Hyperbolic-Graph-Learning#2]
Just as a quick example the command for HyboNet on Tree1111 is not correctly set up:

`python train.py --act=tanh --bias=0 --cuda=-1 --dataset=tree1111_g00_lp --dim=24 --dropout=0.3654186895693169 --epochs=10000 --gamma=0.9 --grad-clip=None --lr=0.4941337733951095 --lr-reduce-freq=750 --manifold=Lorentz --model=HyboNet --momentum=0.6215759601493149 --normalize-feats=0 --num-layers=4 --optimizer=radam --patience=-1 --print-epoch=True --save=0 --save-dir=None --seed=0 --task=lp --weight-decay=0.0012511827632878463 --patience=-1`

The manifold "Lorentz" isn't defined, this is the same for the "HyboNet" model. I wanted to ask if these are just typos (instead of Hyperboloid and HGCN) or if the code base is outdated?

Thank you for any clarification!

**Comment by isaykatsman** [comment]
Nice catch! Sorry about that. That command specifically should be run in the context of the [Fully Hyperbolic Neural Networks repo](https://github.com/chenweize1998/fully-hyperbolic-nn/tree/main). It defines both the model (HyboNet) and the specific isometric manifold implementation of hyperbolic space (Lorentz space). I've just updated the README.
